assets/css/style.css

```diff
@@ -651,203 +631,1812 @@ html {
+  }
+
+  .section-title-container {
+    display: flex;
+    align-items: center;
+    margin-bottom: 0.1vw;
+    width: 90%;
+    margin-left: 6vw;
+  }
+
+  .second-title {
+    font-size: 3.5vw;
+    line-height: 23vw;
+    height: auto;
+    margin-right: 2vw;
+  }
+
+  .line_below_title {
+    flex: 1;
+    height: 1px;
+    background-color: #000;
+    margin: 0;
+    margin-right: 2vw;
+    transform: translateY(-1rem);
+  }
+
+  .card-container {
+    margin-top: -7vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 3vw;
+    margin-bottom: 1vw;
+  }
+
+  .main_card {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 20px;
+    margin-bottom: -19vw;
+  }
+
+  .image-container {
+    display: block;
+    flex-direction: column;
+  }
+
+  .image_from_main_card {
+    width: 44vw;
+    margin: 0;
+  }
+
+  .text-container {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+  }
+
+  .main-card-title {
+    font-size: 3.5vw;
+    margin: 0;
+  }
+
+  .main-card-description {
+    margin-top: 0.8vw;
+    height: auto;
+    font-size: 1.7vw;
+    line-height: 2.8vw;
+  }
+
+  .main-card-list-ul {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  .main_card li {
+    margin-bottom: 0;
+    margin-right: 0;
+    padding: 1vw;
+    width: auto;
+  }
+
+  .main_card ul.main-card-list-ul li.main-card-list-ul-li {
+    padding: 0.6vw 0.7vw;
+    font-size: 1vw;
+    line-height: 1vw;
+    cursor: pointer;
+  }
+
+  .button-from-main-card {
+    font-size: 1.2vw;
+    line-height: 0.3vw;
+    padding: 0.1vw 0.4vw;
+    height: 3.8vw;
+    width: 8.2vw;
+  }
+
+  .works-container {
+    margin-bottom: 5vw;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-left: 2vw;
+    margin-right: 2vw;
+  }
+
+  .work-card {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+    width: 327px;
+    margin: 0;
+  }
+
+  .work-cards-title {
+    font-size: 80%;
+    line-height: 2vw;
+    text-align: start;
+    margin-left: 20px;
+    margin-top: 160px;
+    width: 75%;
+  }
+
+  .work-cards-description {
+    font-size: 55%;
+    line-height: 2vw;
+    text-align: start;
+    margin-left: 20px;
+    width: 88%;
+    margin-top: -22px;
+    margin-bottom: 2px;
+  }
+
+  /* List items */
+  .work-card li {
+    font-size: 40%;
+    line-height: 1vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 21%;
+  }
+
+  .work-card ul {
+    margin-left: 11px;
+    padding: 0;
+    margin-top: 10px;
+  }
+
+  /* Button at the bottom of the card */
+  .button-card {
+    justify-content: center;
+    align-items: center;
+    margin-top: 20px;
+    width: 100%;
+    display: none;
+  }
+
+  .button-card button {
+    font-size: 2vw;
+    line-height: 3vw;
+    cursor: pointer;
+  }
+
```

Comment:
- [ ] As stated in the project requirement you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly implemented **when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project button `appears to the bottom of the image** and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository Just like the Figma Design .

![Capture](https://user-images.githubusercontent.com/93598189/221383638-43dd4534-63df-499f-bcd5-1068df215dd6.PNG)

Your Work card section Look likes this (Kindly adjust like the Figma Design)
![Capture](https://user-images.githubusercontent.com/93598189/221383689-babf7a3b-c9ea-4e51-b0bc-67edd6b2e6b7.PNG)

Reply:
@Abenezer-Tilahun 
I hope this message finds you well. First of all, thank you for your prompt response and for sharing your thoughts with me.

Regarding the implementation of the functionality that requires JavaScript, I want to clarify that I did not include it in my work as per the project's requirements. As per the guidelines, it was explicitly mentioned that there was no need to implement any functionality that requires JavaScript.

While I understand that it may be possible to achieve the same functionality with CSS, I am aware that some might prefer implementing it with JavaScript. However, given the specifications, I didn't explore that option.

If you think that I have misunderstood the requirements, please let me know, and I will be happy to review and incorporate your suggestions.

Thank you for your understanding, and I look forward to hearing back from you soon.

https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md